src/openMVG/multiview/triangulation_test.cpp

```diff
@@ -59,7 +59,8 @@ TEST(Triangulation, TriangulateL1Angular)
     Vec3 X_estimated;
     const Vec2 x1 = d._x[0].col(i);
     const Vec2 x2 = d._x[1].col(i);
-    TriangulateL1Angular(d._R[0], d._t[0], d._K[0].inverse() * x1.homogeneous(), d._R[1], d._t[1], d._K[1].inverse() * x2.homogeneous(), &X_estimated);
+    bool cheir_ok = TriangulateL1Angular(d._R[0], d._t[0], d._K[0].inverse() * x1.homogeneous(), d._R[1], d._t[1], d._K[1].inverse() * x2.homogeneous(), &X_estimated);
```

Comment:
I would use `const bool` to assert we don't change its value before any TEST on it.